components/src/core/drawer/drawer-header/drawer-header.component.ts

```diff
@@ -0,0 +1,34 @@
+import { ChangeDetectionStrategy, Component, Input, ViewEncapsulation } from '@angular/core';
+
+@Component({
+    selector: 'pxb-drawer-header',
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    encapsulation: ViewEncapsulation.None,
+    template: `
+        <mat-toolbar class="pxb-drawer-header">
```

Comment:
refer to the ScoreCard implementation to add a `pxb-score-card-header-background` element in order to allow for background image opacity blending.